language/llama2-70b/user.conf

```diff
@@ -10,4 +10,9 @@
 *.Server.min_duration = 120000
 *.Server.min_query_count = 100
 
-llama2-70b.Server.sample_concatenate_permutation = 1
\ No newline at end of file
+llama2-70b.Server.sample_concatenate_permutation = 1
```

Comment:
All the changes in this file dont't seem needed?

Reply:
Removed